content/docs/api-reference/open.md

```diff
@@ -113,7 +113,7 @@ should handle the event-driven parsing of the document in this case.) This
 increases the performance of the code (minimizing memory usage), and is
 typically faster than loading the whole data into memory.
 
-> If you just needed to load the complete file contents into memory, you can use
+> If you want to load the complete file contents into memory, you can use
```

Comment:
Again, not convinced this is condescending. What's the rationale?

Reply:
Do you agree with that the wording  can be rewritten even more specifically with this change?

Reply:
I agree with you it's not condescending, 

I feel instead of just needed, can't we use -> wanted

Reply:
Even though as the article as says:

## If it's necessary, what is it trying to say?

With documentation, it's rare for terms like easy to be intentionally condescending. Often times, writers use them to show that something isn't as intimidating as it sounds. If this is the case, think about what the docs are trying to communicate and replace the condescending language with word choices that better represent the intention.

If you're struggling to find alternatives, Jim Fisher has some suggestions:

    Be specific: Maybe it's easy because it's quick to set up, doesn't require much typing or has few moving parts.
    Be comparative: Something is smaller than something else. Compared to another product, your product requires less custom configuration.
    Be absolute: It takes 5 lines of code to integrate this library. There are two form fields required.
    Show, don't tell: Instead of using time as an indicator for how easy something is, create a video.

Reply:
Thanks for the reference @kurianbenoy this makes sense cc @andronovhopf you may find this interesting as well 🙂 

But in this case it's neither intentionally nor unintentionally condescending because it's in a conditional sentence so it has a completely different meaning: "if all you need to do is..." and this meaning should be preserved so please roll back or rewrite like in the style I just did. Thanks

Reply:
Still pending to roll this one back please.